clients/src/main/java/org/apache/kafka/common/utils/Time.java

```diff
@@ -30,8 +30,6 @@
  */
 public interface Time {
 
-    Time SYSTEM = new SystemTime();
```

Comment:
If we want to make sure "only one" `SystemTime` is created, maybe we can inline the implementation? Keeping the `Time#SYSTEM` can avoid large changes. For example:

```java
    Time SYSTEM = new Time() {
        @Override
        public long milliseconds() {
            return System.currentTimeMillis();
        }

        @Override
        public long nanoseconds() {
            return System.nanoTime();
        }

        @Override
        public void sleep(long ms) {
            Utils.sleep(ms);
        }

        @Override
        public void waitObject(Object obj, Supplier<Boolean> condition, long deadlineMs) throws InterruptedException {
            synchronized (obj) {
                while (true) {
                    if (condition.get())
                        return;

                    long currentTimeMs = milliseconds();
                    if (currentTimeMs >= deadlineMs)
                        throw new org.apache.kafka.common.errors.TimeoutException("Condition not satisfied before deadline");

                    obj.wait(deadlineMs - currentTimeMs);
                }
            }
        }
    };
```

WDYT?

Reply:
Good idea，i will do it and Check if all ‘new SystemTime()’ can be replaced with ‘Time.SYSTEM’ and Instead of it

Thank you